src/DependencyInjection/Configuration.php

```diff
@@ -58,6 +79,21 @@ private function addResourcesSection(ArrayNodeDefinition $node): void
                                 ->end()
                             ->end()
                         ->end()
+                        ->arrayNode('pixel_event')
```

Comment:
Shouldn't it just be called an event, wdyt?

Reply:
It will be disappointing (Event dispatcher's `Event`, Facebook's Server Side `Event`, this `Event`).
So `PixelEvent` is a good name I believe as it means it is related to our Pixel entity.